flow-client/src/main/java/com/vaadin/client/ResourceLoader.java

```
@@ -639,8 +638,27 @@ public void inlineStyleSheet(String styleSheetContents,
 
             addCssLoadHandler(styleSheetContents, event, styleSheetElement);
 
-            getHead().appendChild(styleSheetElement);
+            addBeforeComment(styleSheetElement, "Stylesheet end");
+        }
+    }
+
+    private void addBeforeComment(Element element, String comment) {
+        elemental.dom.Node commentNode = findComment(getHead(), comment);
```

`commentNode` might be `null` (which signals a bug in the code most likely, because the comment node is expected), so would it be better to fail fast if nothing was found?

---

Added error logging for that case. Sounds better than showing a blank page